drivers/place/spec_helper.cr

```diff
@@ -0,0 +1,7 @@
+module Place; end
```

Comment:
I think if we have this, you should get rid of the clone step in the `spec_helper` file